vertica/datadog_checks/vertica/vertica.py

```diff
@@ -19,13 +20,22 @@
 from .utils import kilobytes_to_bytes, node_state_to_service_check
 
 # Python 3 only
+PROTOCOL_TLS_CLIENT = getattr(ssl, 'PROTOCOL_TLS_CLIENT', ssl.PROTOCOL_TLS)
```

Comment:
I am readding `PROTOCOL_TLS_CLIENT` in case it is used in the base agent.

Reply:
What do you mean ? This is part of the vertica integration, there is no dependency from datadog_checks_base to any check. Relationships are always from the check to datadog_checks_base

Reply:
I originally deleted this in my other PR because I didn't see it being used in the vertica check explicitly, but I wasn't sure if this was being referenced outside of `vertica`, so I added it back in.